automation_tools/satellite6/hammer.py

```diff
@@ -386,3 +386,21 @@ def attach_subscription_to_host_from_satellite(
         organization_id, product_name)
     return hammer('host subscription attach --subscription-id {0} '
                   '--host {1}'.format(subscription_id, hostname))
+
+
+def hammer_determine_cv_and_env_from_ak(ak_name, organization_id):
```

Comment:
By reading this function name I expect it to return both cv and env but just env is being returned. I think the name can be improved to match what is really being determined.